### DISCUSSION TITLE
URN (Uniform Resource Name)

### DISCUSSION THREAD

**Discussion by majid-al-saidi** [codegreencreative/laravel-samlidp#128] [Q&A]
Greetings, is there a way to add the URN (Uniform Resource Name) to the attributes? 
I tried this:
`->addAttribute(new Attribute('attribute', $value, 'urn:oid:X.X.X.X'))`

The result:
`<Attribute Name="attribute">
    <AttributeValue>$value</AttributeValue>
   </Attribute>`

There is no urn.

**Comment by upwebdesign** [comment]
@majid-al-saidi, I believe this is what you are looking for:
```php
->addAttribute(new Attribute('attribute', 'urn:oid:X.X.X.X'));
```
